host_applications/linux/apps/raspicam/RaspiCamControl.c

```diff
@@ -1811,6 +1813,22 @@ void default_camera_control_callback(MMAL_PORT_T *port, MMAL_BUFFER_HEADER_T *bu
                         settings->awb_red_gain.num, settings->awb_red_gain.den,
                         settings->awb_blue_gain.num, settings->awb_blue_gain.den);
       }
+      case MMAL_PARAMETER_CAPTURE_STATUS:
+      {
+         wiringPiSetup();
+         pinMode(21, OUTPUT);
+         MMAL_PARAMETER_CAPTURE_STATUS_T *status = (MMAL_PARAMETER_CAPTURE_STATUS_T*)param;
+         if (status->status == MMAL_PARAM_CAPTURE_STATUS_CAPTURE_STARTED)
+         {
+            vcos_log_error("Started capture. Setting pin 21 (BCM pin 5) high...");
```

Comment:
I'm not sure, but I think 6by9 used this `vcos_log_error` function because it guarantees this gets printed to the console. I haven't looked into whether there are better logging methods (and I don't really think it's worth it)

Reply:
whatevs. this seems nice.
this probably doesn't end up in the run_experiment log file, does it? if so, cool. if not, nbd